src/test/java/org/openrewrite/java/testing/junit5/RemoveTryCatchBlocksFromUnitTestsTest.java

```diff
@@ -0,0 +1,145 @@
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion()
+            .classpathFromResources(new InMemoryExecutionContext(),
+                                    "junit-jupiter-api-5.9",
+                                    "junit-jupiter-params-5.9",
+                                    "junit-4.13"))
+          .recipe(new RemoveTryCatchBlocksFromUnitTests());
+    }
+
+    @Test
+    @DocumentExample
+    void removeTryCatchBlock() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import org.junit.Assert;
+              import org.junit.jupiter.api.Test;
+              
+              class MyTest {
+                  @Test
+                  public void testMethod() {
+                      try {
+                          int divide = 50 / 0;
+                      }catch (ArithmeticException e) {
+                          Assert.fail(e.getMessage());
+                      }
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.api.Assertions;
+              import org.junit.jupiter.api.Test;
+              
+              class MyTest {
+                  @Test
+                  public void testMethod() {
+                      Assertions.assertDoesNotThrow(() -> {
+                          int divide = 50 / 0;
+                      });
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void onlyAffectsUnitTests() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import org.junit.Assert;
+              
+              class Test {
+                  public void method() {
+                      try {
+                          int divide = 50/0;
+                      }catch (ArithmeticException e) {
+                          Assert.fail(e.getMessage());
+                      }
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void statementsBeforeAndAfterTryBlock() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import org.junit.Assert;
+              import org.junit.jupiter.api.Test;
+              
+              class MyTest {
+                  @Test
+                  public void testMethod() {
+                      System.out.println("statements before");
+                      int x = 50;
+                      try {
+                          int divide = 50 / 0;
+                      }catch (ArithmeticException e) {
+                          Assert.fail(e.getMessage());
+                      }
+                      System.out.println("statements after");
+                      int y = 50;
+                      int z = x + y;
+                  }
+              }
+              """,
+            """
+              import org.junit.Assert;
```

Comment:
By removing the `visitMethodDeclaration()` override as I suggested, I only had to fix this one import to get the test to pass.
```suggestion
              import org.junit.jupiter.api.Assertions;
```